tasks/cockpit-tasks-webhook.yaml

```diff
@@ -40,6 +40,10 @@ items:
             - name: webhook-secrets
               mountPath: /run/secrets/webhook
               readOnly: true
+            env:
+            - name: RELEASE_SINK
+              value: sink
+          - name: AMQP_SERVER
```

Comment:
This is an error I think.

Plus the webhook doesn't recognize `AMQP_SERVER` at this point, though that would probably be a good improvement.

Reply:
Whoops, thanks for spotting! Fixed. (copy&paste error indeed).

I didn't dare to re-deploy the webhook yet, as I just pulled out CI from a huge swamp :/